### ISSUE TITLE
How is the model trained when calculating the test RMSE? Is it trained for 100 epochs without using a validation set?

### ISSUE THREAD

**Issue by nibingwei123** [molML/MoleculeACE#15]

**Comment by nibingwei123** [comment]
Hello, I would like to know how the test RMSE in the MoleculeACE_results.csv file was obtained. Was it obtained by training for x epochs, where x is the number of epochs determined by five-fold cross-validation, without a validation set?
